packages/wonder-blocks-layout/src/components/strut.js

```diff
@@ -14,7 +16,12 @@ type Props = {|
  */
 export default class Strut extends React.Component<Props> {
     render(): React.Node {
-        return <View aria-hidden="true" style={strutStyle(this.props.size)} />;
+        const {size, style} = this.props;
+        return (
+            <View style={style}>
+                <View aria-hidden="true" style={strutStyle(size)} />
+            </View>
```

Comment:
It be preferable to return only a single `View`.  Is there a reason why `style` and `strutStyle(size)` can't both be set on a single `View`?

Reply:
Yeah, it looks like Spring is a single element, so I think we should change this to match.

Reply:
Hey guys. Thanks for bringing this up. I did originally have `Strut` be a single `View`, but it seemed that it was not applying any color from `style`. Now that i'm further looking into to it, it was because single-`View` `Strut` would have a natural height of zero if no height was specified. I got that fixed 👍🏻